### ISSUE TITLE
sklearn - Chen Yulin's Blog

### ISSUE THREAD

**Issue by Chen-Yulin** [Chen-Yulin/Chen-Yulin.github.io#28]
https://chen-yulin.github.io/2023/05/08/obsidian/Python/sklearn/ 

 Some basic operationShuffle the training set for hold out validation12from sklearn.utils import shuffletraining_set,dev_set &#x3D; np.split(shuffle(data_sample_35),[25]) Also capable of spliting training s
